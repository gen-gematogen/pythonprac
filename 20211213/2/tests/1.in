[956, 910, 780, 704, 625, 614, 601, 525, 430, 239, 36, 31, 24, 15, 8, 1]
