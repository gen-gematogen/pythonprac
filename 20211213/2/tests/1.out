[1, 8, 15, 24, 31, 36, 239, 430, 525, 601, 614, 625, 704, 780, 910, 956]
